notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1165_m1m3_bump_test_error_and_measured_forces.ipynb

```diff
@@ -5,19 +5,12 @@
    "execution_count": null,
```

Comment:
<p>Hi Craig, </p><p> </p><p>Thanks a lot for adding that analysis. For completeness, it would be nice if you could add a section summarizing your finding, similar to the conclusions you added to the jira ticket: </p><p> </p><p><em>In terms of the statistics of the failures, we have:</em></p><ul><li><em>actual force overshoot compared to the demanded force: about 23%</em></li><li><em>actual force undershoot compared to the demanded force: about 74%</em></li><li><em>excessive latency of the actual force compared to the demanded force: None seen</em></li><li><em>locked/constant force independent of demand: about 6%</em></li></ul><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>